src/lib/src/receiver.cpp

```diff
@@ -160,8 +152,10 @@ int roc_receiver_close(roc_receiver* receiver) {
 
     roc_context& context = receiver->context;
 
-    receiver->receiver.iterate_ports(receiver_close_port, receiver);
-    receiver->context.allocator.destroy(*receiver);
+    for (size_t i = 1; i < receiver->addresses.size(); i++) {
```

Comment:
Why do we start iterating from `1`?

Reply:
Sorry my mistake, I was tired yesterday. And of course it fixes the problem i've encountered in the other branch

Reply:
No problem :) Thanks!